src/main.rs

```diff
@@ -19,6 +21,8 @@ use simplelog::{LevelFilter, SimpleLogger};
 const TERMINAL: &str = "terminator";
 const LAUNCHER: &str = "dmenu_run";
 
+const AUTOSTART: [&str; 0] = [];
```

Comment:
Please add a comment above that states that this is for the autostarting of applications, and to see configure.md